depstubber.go

```diff
@@ -48,7 +51,24 @@ func main() {
 	}
 
 	if *modeAutoDetection {
-		pathToTypeNames, pathToFuncAndVarNames, err := autoDetect(".", ".")
+		{
+			wd, err := os.Getwd()
+			if err != nil {
+				log.Fatalf("Unable to load current director: %v", err)
+			}
+			{ // Remove current /vendor dir if exists:
```

Comment:
Not sure why you're doing this, but I'd prefer to at least ask the user before the tool performs any potentially destructive operations.

Reply:
I can add a question:

```bash
A ./vendor folder already exists. Would you like it to be deleted? [Y/n]
```
And a `-y` flag to not ask the question (sets to yes).